### ISSUE TITLE
Install buttons on discovery pane don't show up in high contrast mode

### ISSUE THREAD

**Issue by aswan** [mozilla/addons-frontend#4662]
### Describe the problem and steps to reproduce it:
Put the browser into high contrast mode (Preferences → General → Colors → Override the colors specified by the page with your selections above "Always") and visit the discovery pane

### What happened?
The interior part of the install buttons on the discovery pane are not visible:
https://bug1445552.bmoattachments.org/attachment.cgi?id=8958689

### What did you expect to happen?
For them to be visible I suppose

### Anything else we should know?
this was originally filed at https://bugzilla.mozilla.org/show_bug.cgi?id=1445552

**Comment by AlexandraMoga** [comment]
@bobsilverberg As far as I can see, AMO is not compatible with high contrast mode either. Do we want too address this too?

https://www.screencast.com/t/xVwau2y37

**Comment by tofumatt** [comment]
I would like for this to be a P2 but I don't think it blocks release or is more crucial than a lot of features we'll be landing that affect more users. 😞

**Comment by willdurand** [comment]
This should be fixed by the new install button.

**Comment by AlexandraMoga** [comment]
The new install buttons are visible and functional on the discovery pane in high contrast mode:

https://www.screencast.com/t/0zxoqg4m

Marking issue as verified fixed
